forge-gui/res/cardsfolder/w/wrap_in_vigor.txt

```diff
@@ -1,5 +1,5 @@
 Name:Wrap in Vigor
 ManaCost:1 G
 Types:Instant
-A:SP$ RegenerateAll | ValidCards$ Creature.YouCtrl | SpellDescription$ Regenerate each creature you control.
+A:SP$ RegenerateAll | Defined$ Valid Creature.YouCtrl | SpellDescription$ Regenerate each creature you control.
```

Comment:
Missed this one (`RegenerateAll` --> `Regenerate`), if I'm reading the changes correctly and `RegenerateAll` was completely removed.